Rakefile

```diff
@@ -0,0 +1,20 @@
+require "rake"
+
+task :default => :build
+
+# rake build
+desc "Build the site"
+task :build do
+  shell_args = %w[bundle exec jekyll build]
+  sh *shell_args
```

Comment:
Yes, I should inline it. I copy it from the Rakefile of my own website. So forget to do so. 

> Does Jekyll have any Rake helpers we can use?

Not that I am aware of.